### PR TITLE
Use CARGO_TARGET_DIR for target directory

### PR DESCRIPTION
**Pull request by L3nn0x**
Hello, if the target directory is not `target` then grcov will fail to run. This is a patch to correct this behavior.

Let me know if there is anything I need to do to get this merged in.

Thanks!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "grcov"
-version = "0.8.18"
```
Comment (marco-c) [inline Cargo.toml:3]: Could you revert this change? I'll bump the version in a later patch

Comment (L3nn0x) [reply Cargo.toml]: done!

**Comment by marco-c** [comment]
There is a linting error related to formatting, could you run `cargo fmt`?

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**